README.md

```
@@ -118,6 +181,15 @@ Data is saved to `data/openaq/processed/{country}_airquality_all_{timestamp}.csv
 - `unit`: Measurement unit
 - `city`, `country`: Geographic info
 
+**Traffic Data:**
+- `timestamp`: UTC timestamp
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)